apstools/utils/misc.py

```diff
@@ -546,7 +546,7 @@ def connect_pvlist(pvlist, wait=True, timeout=2, poll_interval=0.1):
         obj_dict[oname] = obj
 
     if wait:
-        times_up = time.time() + min(0, timeout)
+        times_up = time.time() + max(0, timeout)
         poll_interval = min(0.01, poll_interval)
```

Comment:
It's not directly related to the bug in question, but it looks like this should also be changed from min to max.

Reply:
Good catch!